### DISCUSSION TITLE
Welcome to pymssql Discussions!

### DISCUSSION THREAD

**Discussion by termim** [pymssql/pymssql#736] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  P.S. Bugs should still be reported as issues

**Comment by klaymartens** [comment]
Are you guys aware your read the docs site has been hacked? Bunch of dicey links placed in there....dunno if you are still maintaining it, but you may want to take a look.....
